code/game/dna/dna2.dm

```diff
@@ -189,6 +190,38 @@ GLOBAL_LIST_EMPTY(bad_blocks)
 		val=rand(1, 2049)
 	SetUIValue(block, val, defer)
 
+//Get Tri State Block State
+/datum/dna/proc/GetUITriState(block)
+	if(block <= 0)
+		return
+	var/val = GetUIValue(block)
+	switch(val)
+		if(1395 to 2760)
+			return 0
```

Comment:
What is 0,1,2 in this context? Magic numbers are less than good, so these should really be defined somewhere in `code/__DEFINES`, like so

```dm
#define WHATEVER_0_IS 0
#define WHATEVER_1_IS 1
#define WHATEVER_2_IS 2
```

Reply:
Defined as DNA_GENDER_FEMALE, DNA_GENDER_FEMALE, DNA_GENDER_PLURAL.